mcs_benchmark_data/pipelines/commonsense_qa/kagnet_commonsense_qa_submission_pipeline.py

```diff
@@ -13,19 +13,20 @@
 
 
 class KagnetCommonsenseQaSubmissionPipeline(_Pipeline):
-    __ID = "CommonsenseQA"
+    ID = "CommonsenseQA"

```

Comment:
Should probably reconsider pipeline_id as benchmark_id and what you're calling SUBMISSION_NAME as submission_id.

Reply:
In paradicms the pipeline_id is used to locate the `data/` subdirectory. You don't need that here, you're using `benchmark_id` and possibly `submission_id`.